# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Chester Cheng <ccheng@redhat.com>, 2012-2013
# Terry Chuang <tchuang@redhat.com>, 2011,2013
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-04-08 15:09+0200\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2015-03-21 07:06-0400\n"
"Last-Translator: Cheng-Chia Tseng <pswo10680@gmail.com>\n"
"Language-Team: Chinese (Taiwan) (http://www.transifex.com/projects/p/gnome-"
"abrt/language/zh_TW/)\n"
"Language: zh-TW\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Zanata 3.7.1\n"

#: ../src/gnome-abrt:265
msgid "My"
msgstr "我的"

#: ../src/gnome-abrt:268 ../src/gnome_abrt/views.py:135
msgid "System"
msgstr "系統"

#: ../src/gnome-abrt:288
msgid "_Preferences"
msgstr "偏好設定(_P)"

#: ../src/gnome-abrt:289
msgid "_ABRT Configuration"
msgstr "ABRT 配置(_A)"

#: ../src/gnome-abrt:290
msgid "_Report problem with ABRT"
msgstr "透過 ABRT 回報問題(_R)"

#: ../src/gnome-abrt:294
msgid "_About"
msgstr "關於(_A)"

#: ../src/gnome-abrt:295
msgid "_Quit"
msgstr "離開(_Q)"

#: ../src/gnome-abrt.desktop.in.h:1 ../src/gnome_abrt/views.py:257
msgid "Problem Reporting"
msgstr "問題回報"

#: ../src/gnome-abrt.desktop.in.h:2
msgid "View and report application crashes"
msgstr "檢視與回報應用程式錯誤"

#: ../src/gnome-abrt.appdata.xml.in.h:1
msgid ""
"Collection of software tools designed for collecting, analyzing and "
"reporting of software issues."
msgstr "用於蒐集、分析與回報軟體問題的軟體工具之集合。"

#: ../src/gnome-abrt.appdata.xml.in.h:2
msgid ""
"Its main purpose is to ease the process of reporting an issue and finding a "
"solution."
msgstr "主要用途是簡化回報問題的程序，並找尋解決方案。"

#: ../src/gnome_abrt/dbus_problems.py:89
#, python-brace-format
msgid "Can't add receiver of signal '{0}'on DBus system path '{1}': {2}"
msgstr "無法在 DBus 系統路徑 '{1}' 上新增訊號為 '{0}' 的接收程式：{2}"

#: ../src/gnome_abrt/dbus_problems.py:102
#, python-brace-format
msgid "Can't connect to DBus system bus '{0}' path '{1}': {2}"
msgstr "無法連上 DBus 系統 bus '{0}' 路徑 '{1}'：{2}"

#: ../src/gnome_abrt/dbus_problems.py:109
#, python-brace-format
msgid "Can't get interface '{0}' on path '{1}' in bus '{2}': {3}"
msgstr "無法取得介面 '{0}' 於路徑 '{1}'，位於匯流排 '{2}'：{3}"

#: ../src/gnome_abrt/dbus_problems.py:154
msgid "Can't chown problem '{0}' over DBus service: {1!s}"
msgstr "無法透過 DBus 服務 chown 問題「{0}」：{1!s}"

#: ../src/gnome_abrt/dbus_problems.py:170
msgid "Can't get problem data from DBus service: {0!s}"
msgstr "無法從 DBus 服務取得問題資料: {0!s}"

#: ../src/gnome_abrt/dbus_problems.py:181
msgid "Can't get list of problems from DBus service: {0!s}"
msgstr "無法從 DBus 服務去得問題清單：{0!s}"

#: ../src/gnome_abrt/dbus_problems.py:195
msgid "Can't delete problem over DBus service: {0!s}"
msgstr "無法在 DBus 服務上刪除問題：{0!s}"

#: ../src/gnome_abrt/dialogs.py:55
msgid "The description of problem is too short"
msgstr "問題的描述太短"

#: ../src/gnome_abrt/dialogs.py:56
msgid ""
"In order to get more useful reports we do not accept reports with the "
"description shorter than 10 letters."
msgstr "為了得到更有幫助的回報，我們不接受描述少於 10 個字母的錯誤報告。"

#: ../src/gnome_abrt/dialogs.py:66
msgid "The summary of problem is too short"
msgstr "問題的概要太短"

#: ../src/gnome_abrt/dialogs.py:67
msgid ""
"In order to get more useful reports we do not accept reports with the "
"summary shorter than 10 letters."
msgstr "為了得到更有幫助的回報，我們不接受概要少於 10 個字母的錯誤報告。"

#: ../src/gnome_abrt/dialogs.py:93
msgid "Problem description"
msgstr "問題描述"

#: ../src/gnome_abrt/dialogs.py:101
msgid "Summary:"
msgstr "摘要："

#: ../src/gnome_abrt/dialogs.py:120
msgid "_Send"
msgstr "傳送(_S)"

#. Sets text but text is selected :)
#: ../src/gnome_abrt/dialogs.py:134
msgid "A problem with ABRT"
msgstr "ABRT 的問題"

#: ../src/gnome_abrt/directory_problems.py:216
#, python-brace-format
msgid ""
"You have probably reached inotify's limit on the number of watches in '{0}'. "
"The limit can be increased by proper configuration of inotify. For more "
"details see man inotify(7). This event causes that you will not be notified "
"about changes in problem data happening outside of this application. This "
"event do not affect any other functionality."
msgstr ""
"您可能已達 inotify 在 '{0}' 中的監控計數限制。這項限制可藉由正確配置 inotify 來增加。欲取得更多資訊，請參閱 man "
"inotify(7)。這項事件會使您不被通知有關於此應用程式範圍外的問題資料上的變更。這項事件不會影響任何其它功能。"

#: ../src/gnome_abrt/directory_problems.py:262
#, python-brace-format
msgid "Can't open directory: '{0}'"
msgstr "無法開啟目錄：'{0}'"

#: ../src/gnome_abrt/oops.glade.h:1
msgid "_Delete"
msgstr "刪除(_D)"

#: ../src/gnome_abrt/oops.glade.h:2
msgid "Delete selected problems"
msgstr "刪除所選擇的問題"

#: ../src/gnome_abrt/oops.glade.h:3
msgid "_Report"
msgstr "回報(_R)"

#: ../src/gnome_abrt/oops.glade.h:4
msgid "Submits selected problem"
msgstr "提交所選取的問題"

#: ../src/gnome_abrt/oops.glade.h:5
msgid "Analy_ze"
msgstr "分析(_z)"

#: ../src/gnome_abrt/oops.glade.h:6
msgid "Open selected problem for analysis"
msgstr "開啓選擇的問題以進行分析"

#: ../src/gnome_abrt/oops.glade.h:7
msgid "D_etails"
msgstr "細節(_E)"

#: ../src/gnome_abrt/oops.glade.h:8
msgid "Show technical details"
msgstr "顯示技術細節"

#: ../src/gnome_abrt/oops.glade.h:9
msgid "_All problems"
msgstr "所有問題（_A）"

#: ../src/gnome_abrt/oops.glade.h:10
msgid "Show all known problems from all system users"
msgstr "從所有系統使用者顯示所有已知問題"

#: ../src/gnome_abrt/oops.glade.h:11
msgid "_Open problem's data directory"
msgstr "開啓問題的資料目錄(_O)"

#: ../src/gnome_abrt/oops.glade.h:12
msgid ""
"Opens the selected problem's data directory in the default file system "
"browser"
msgstr "在預設的檔案系統瀏覽器中開啓已選擇的問題的資料目錄"

#: ../src/gnome_abrt/oops.glade.h:13
msgid "_Copy problem's ID to Clipboard"
msgstr "將問題的 ID 複製至剪貼簿(_C)"

#: ../src/gnome_abrt/oops.glade.h:14
msgid "Stores the selected problem's ID in Clibpoard"
msgstr "將選擇的問題之 ID 儲存在剪貼簿中"

#: ../src/gnome_abrt/oops.glade.h:15
msgid "_Filter"
msgstr "過濾(_F)"

#: ../src/gnome_abrt/oops.glade.h:16
msgid "Show filter box"
msgstr "顯示過濾方塊"

#: ../src/gnome_abrt/oops.glade.h:17
msgid "Name"
msgstr "名稱"

#: ../src/gnome_abrt/oops.glade.h:18
msgid "Version"
msgstr "版本"

#: ../src/gnome_abrt/oops.glade.h:19
msgid "Detected"
msgstr "已偵測到"

#: ../src/gnome_abrt/oops.glade.h:20
msgid "<a href=\"https://glade.gnome.org/\">Very long placeholder URL tile</a>"
msgstr "<a href=\"https://glade.gnome.org/\">非常長的佔位用 URL 鋪磚</a>"

#: ../src/gnome_abrt/oops.glade.h:21
msgid "<a href=\"https://github.com/abrt/gnome-abrt\">Placeholder URL</a>"
msgstr "<a href=\"https://github.com/abrt/gnome-abrt\">佔位用 URL</a>"

#: ../src/gnome_abrt/oops.glade.h:22 ../src/gnome_abrt/views.py:875
msgid "Reported"
msgstr "已回報"

#: ../src/gnome_abrt/oops.glade.h:23
msgid "button"
msgstr "按鈕"

#: ../src/gnome_abrt/oops.glade.h:24
msgid "No problems detected!"
msgstr "沒有偵測到問題！"

#: ../src/gnome_abrt/oops.glade.h:25
msgid "No source selected!"
msgstr "未選擇來源！"

#: ../src/gnome_abrt/oops.glade.h:26
msgid "No source"
msgstr "沒有來源"

#: ../src/gnome_abrt/problems.py:467
#, python-brace-format
msgid "Not found in cache but deleted: {0}"
msgstr "在快取中找不到，已刪除：{0}"

#: ../src/gnome_abrt/tools.py:34
msgid "Future"
msgstr "未來"

#: ../src/gnome_abrt/tools.py:41
msgid "Yesterday"
msgstr "昨天"

#: ../src/gnome_abrt/tools.py:50
#, python-brace-format
msgid "Last week"
msgid_plural "{0:d} weeks ago"
msgstr[0] ""

#: ../src/gnome_abrt/tools.py:53
#, python-brace-format
msgid "Last month"
msgid_plural "{0:d} months ago"
msgstr[0] ""

#: ../src/gnome_abrt/tools.py:56
#, python-brace-format
msgid "Last year"
msgid_plural "{0:d} years ago"
msgstr[0] ""

#: ../src/gnome_abrt/views.py:283
msgid "Failed to load UI definition"
msgstr "載入 UI 定義失敗"

#: ../src/gnome_abrt/views.py:818
msgid "Unexpected system error"
msgstr "未預期的系統錯誤"

#: ../src/gnome_abrt/views.py:820
msgid "The system has encountered a problem and recovered."
msgstr "系統遭遇到問題，並從中恢復過來。"

#: ../src/gnome_abrt/views.py:823
msgid "Fatal system failure"
msgstr "嚴重系統錯誤"

#: ../src/gnome_abrt/views.py:825
msgid "The system has encountered a problem and could not continue."
msgstr "系統遭遇到問題，並且無法繼續。"

#: ../src/gnome_abrt/views.py:838
#, python-brace-format
msgid "{0} problem has been detected"
msgstr "找到 {0} 個問題"

#: ../src/gnome_abrt/views.py:842
#, python-brace-format
msgid "{0} quit unexpectedly"
msgstr "{0} 未預期退出"

#: ../src/gnome_abrt/views.py:845
msgid "The application encountered a problem and could not continue."
msgstr "應用程式遭遇到問題，並且無法繼續。"

#: ../src/gnome_abrt/views.py:848 ../src/gnome_abrt/views.py:850
msgid "N/A"
msgstr "N/A"

#: ../src/gnome_abrt/views.py:878
msgid "cannot be reported"
msgstr "無法回報"

#: ../src/gnome_abrt/views.py:883
msgid "Reports"
msgstr "報告"

#: ../src/gnome_abrt/views.py:889
msgid ""
"This problem has been reported, but a <i>Bugzilla</i> ticket has not been "
"opened. Our developers may need more information to fix the problem.\n"
"Please consider also <b>reporting it</b> to Bugzilla in order to provide "
"that. Thank you."
msgstr ""
"此問題已經回報過，但卻沒有打開 <i>Bugzilla</i> 請票。我們的開發人員也許需要更多資訊才能修正這個問題。\n"
"請您考慮將之<b>回報</b>到 Bugzilla 以提供相關資訊。感謝您。"

#: ../src/gnome_abrt/views.py:894
msgid "yes"
msgstr "是"

#: ../src/gnome_abrt/views.py:896
msgid "no"
msgstr "無"

#: ../src/gnome_abrt/controller.py:78
msgid ""
"Failed to execute the requested action because of an error in new process "
"creation"
msgstr "因為建立新程序時產生問題，故無法執行所要求的動作"
